packages/eslint-plugin-pf-codemods/lib/rules/v5/table-rename-TableComposable.js

```diff
@@ -0,0 +1,97 @@
+const {getPackageImports} = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8892
+module.exports = {
+  meta: {fixable: "code"},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-table");
+    const tableComposableImport = imports.find(
+      (imp) => imp.imported.name === "TableComposable"
+    );
+
+    return imports.length === 0 || !tableComposableImport
+      ? {}
+      : {
+          // update opening/closing elements
+          JSXIdentifier(node) {
+            const nodeName = node.name;
+            const importedNode = imports.find(
+              (imp) => imp.local.name === nodeName
+            );
+            const isAliased =
+              importedNode?.local?.name !== importedNode?.imported?.name;
+            if (
+              importedNode?.imported?.name === "Table" &&
+              importedNode?.local?.name === nodeName
+            ) {
+              // if data-codemods attribute, do nothing
+              const parentNode = node.parent;
+              const isOpeningTag = parentNode.type === "JSXOpeningElement";
+              const openingTagAttributes = isOpeningTag
+                ? parentNode.attributes
+                : parentNode.parent.openingElement.attributes;
+              const hasDataAttr =
+                openingTagAttributes &&
+                openingTagAttributes.filter(
+                  (attr) => attr.name?.name === "data-codemods"
+                ).length;
+              if (hasDataAttr) {
+                return;
+              }
+            }
+            if (
+              nodeName === "TableComposable" ||
+              nodeName !== tableComposableImport.local.name
+            ) {
+              const tableName = nodeName.replace("TableComposable", "Table");
+              const needsDataAttr = tableName === "Table" || isAliased;
+              const addDataAttr = (jsxStr) =>
+                `${jsxStr.slice(0, -1)} data-codemods="true">`;
```

Comment:
should we also consider self closing tags? this wouldn't work for those.. perhaps
```suggestion
                `${jsxStr.replace(/(\/)*>$/, ' data-codemods="true" $1>`;
```